src/RuleSet.php

```diff
@@ -165,22 +165,22 @@
                 'before_array_assignments_colon' => false,
             ],
         ],
-        '@PHP56Migration' => [
+        '@PHP56Migration:risky' => [
             'pow_to_exponentiation' => true,
         ],
         '@PHP70Migration' => [
-            '@PHP56Migration' => true,
+            'ternary_to_null_coalescing' => true,
+        ],
+        '@PHP70Migration:risky' => [
+            '@PHP56Migration:risky' => true,
+            'declare_strict_types' => true,
             'non_printable_character' => [
                 'use_escape_sequences_in_strings' => true,
```

Comment:
Not that with this option the `non_printable_character` fixer is not risky so having it in the `@PHP70Migration` ruleset is valid, though this will make the test fail as the `isRisky()` method always returns `true`, so not sure what to do here.

Reply:
I don't understand how - looking at [this](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.5/tests/Fixer/Basic/NonPrintableCharacterFixerTest.php#L151) test case both string are different and someone might relate to comparing with such a string and fixer will change it (it as bad as overriding native functions).

Reply:
Those strings are not different (on PHP 7+), see https://3v4l.org/MMMlM.

Reply:
Oh, yeah, right, I made mistake in mt test - silly me... So what about it? I'd leave it here - in README it's described as risky, so I'd expect it to belong to only risky groups.